### PR TITLE
Fix  CMake error for CMake >=3.30

### PR DESCRIPTION
**Pull request by baijumeswani**
Mirror of https://github.com/microsoft/onnxruntime-extensions/pull/754

### CODE REVIEW

**Review by snnn — APPROVED** [review]

**Comment by snnn** [comment]
Would you mind also doing it for the main ORT repo?

**Review by yufenglee — APPROVED** [review]

File: src/python/CMakeLists.txt [hunk 1]
```diff
@@ -25,6 +25,12 @@ if(USE_CUDA AND CMAKE_CUDA_COMPILER)
   target_link_libraries(python PRIVATE cublasLt cublas cudnn curand cufft cudart)
 endif()
 
+# Avoid warning of Calling FetchContent_Populate(Lib) is deprecated temporarily
```
Comment (yufenglee) [inline src/python/CMakeLists.txt:28]: what is the new function to replace it?

Comment (snnn) [reply src/python/CMakeLists.txt:28]: fetchcontent_makeavailable

Comment (baijumeswani) [reply src/python/CMakeLists.txt:28]: https://cmake.org/cmake/help/latest/policy/CMP0169.html has more details.

**Pull request merged**